internal/backend/local/testing.go

```diff
@@ -202,27 +202,31 @@ func mustResourceInstanceAddr(s string) addrs.AbsResourceInstance {
 }
 
 // assertBackendStateUnlocked attempts to lock the backend state. Failure
```

Comment:
I liked how you added "for a test" below, would you mind adding that here?

Reply:
Good catch, I meant to update it as well